control.js

```diff
@@ -1,77 +1,108 @@
-
-$(function(){
+$(function() {
+  
   //play the video ro pause the video
-  $("#playOrPause").click(function(){
-      var video = byId("video");
-     if(video.paused){video.play();}
-     else video.pause();
+  $("#playOrPause").click(function() {
+    var video = byId("video");
+    if(video.paused) {
+      video.play();
+      $("#playOrPause").css({
+        "border":"3px inset grey"
+      });
+    } else {
+       $("#playOrPause").css({
+        "border":"3px outset gray"
+      });
+      video.pause();
+    }
   });
   
   //stop the video 
-  $("#stop").click(function(){
-     var video = byId("video");
+  $("#stop").click(function() {
+    var video = byId("video");
     video.load();
     video.pause();
+    
   });
+  
   //reload the video 
-  $("#reload").click(function(){
-     var video = byId("video");
+  $("#reload").click(function() {
+    var video = byId("video");
     video.load();
     video.play();
   });
   
```

Comment:
whether can optimize code for video control function, just as make the video for a member variable, avoid repeat get it  or use one function by parameter